### PR TITLE
Add new API documentation

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/api.rst [hunk 1]
```diff
@@ -6,4 +6,18 @@ JavaScript API
 Any c2cgeoportal application exposes a JavaScript API. This API can be used in third-party applications,
 like CMS applications.
 
-.. TODO: fill for new API
+In the folder ``geoportal/<package>_geoportal/static-ngeo/api/apihelp/`` we have all the files related to
```
Comment (jwkaltz) [inline doc/integrator/api.rst]: Suggestion:
The API help is in the folder ``geoportal/<package>_geoportal/static-ngeo/api/apihelp/``.

File: doc/integrator/api.rst [hunk 2]
```diff
@@ -6,4 +6,18 @@ JavaScript API
...
+the API help.
+
+In the ``geoportal/demo_geoportal/static-ngeo/api/api.css`` file you have the API CSS.
```
Comment (jwkaltz) [inline doc/integrator/api.rst]: In the file ``geoportal/demo_geoportal/static-ngeo/api/api.css``, you have the API CSS.

File: doc/integrator/api.rst [hunk 3]
```diff
@@ -6,4 +6,18 @@ JavaScript API
...
+
+In the ``geoportal/demo_geoportal/static-ngeo/api/index.js`` wou can customise you API with:
```
Comment (jwkaltz) [inline doc/integrator/api.rst]: In the file ``geoportal/demo_geoportal/static-ngeo/api/index.js``, you can customize your API with:

File: doc/integrator/api.rst [hunk 4]
```diff
@@ -6,4 +6,18 @@ JavaScript API
...
+
+.. code:: JavaScript
+
+   constants.themesUrl: '<the theme URL>';
+   constants.projection: <the projection>;
+   constants.supportedProjections: [<the projection>, <an other projection>];
```
Comment (jwkaltz) [inline doc/integrator/api.rst]: <another projection>

Comment (sbrunner) [reply doc/integrator/api.rst]: Empty comment?

**Comment by sbrunner** [comment]
Thanks @jwkaltz for your review :-)

File: doc/integrator/api.rst (hunk 4 above)
Comment (jwkaltz) [reply doc/integrator/api.rst]: Hmm. I wanted to comment:
another projection
(instead of an other projection)

Comment (sbrunner) [reply doc/integrator/api.rst]: +1 thanks :-)

**Pull request merged**